docs/sources/installation/mac.md

```diff
@@ -8,14 +8,13 @@ You can install Docker using Boot2Docker to run `docker` commands at your comman
 Choose this installation if you are familiar with the command-line or plan to
 contribute to the Docker project on GitHub.
 
+[<img src="/installation/images/kitematic.png" alt="Download Kitematic"
```

Comment:
there is a reason :) mkdocs will correctly link up the markdown to the `/kitematic/` at the `.md` file level - whereas using the html link, mkdocs ignores is as a raw html link.